xee/ext.py

```diff
@@ -147,6 +152,7 @@ def open(
       ee_init_kwargs: Optional[Dict[str, Any]] = None,
       ee_init_if_necessary: bool = False,
       executor_kwargs: Optional[Dict[str, Any]] = None,
+      tile_fetch_kwargs: Optional[Dict[str, int]] = None,
```

Comment:
It's going to be really difficult to undo the naming of this arg. I think a better name might be one that's more explicit: `getitem_kwargs`. This arg serves one purpose only, which is forwarding params to the `robust_getitem`. We should name it as such.

I'm really sorry about the churn. Let me know if you'd like me to make this change and I'd be happy to.

Reply:
Yes I likes this new name `getitem_kwargs`.